ocp_resources/pod_metrics.py

```diff
@@ -0,0 +1,11 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class PodMetrics(NamespacedResource):
+    """
+    PodMetrics object. API reference:
+    This resource is managed by: https://github.com/kubernetes-sigs/metrics-server and is a built-in resource
+    """
+
+    api_group = NamespacedResource.ApiGroup.METRICS_K8S_IO
+    api_version = NamespacedResource.ApiVersion.V1BETA1
```

Comment:
api_version is not needed, we get it from the cluster.

Reply:
Done.